code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -190,6 +190,28 @@
 	color = "#454545"
 	list_reagents = list(/datum/reagent/mutationtoxin/shadow = 5)
 
+///////////////////////////////////////// Psychologist inventory pills
+/obj/item/reagent_containers/pill/happinesspsych
+	name = "mood stabilizer pill"
+	desc = "Used to temporarily alleviate anxiety and depression, take only as prescribed."
+	list_reagents = list(/datum/reagent/drug/happiness = 5)
+	icon_state = "pill_happy"
+	rename_with_volume = TRUE
+
+/obj/item/reagent_containers/pill/paxpsych
+	name = "pacification pill"
+	desc = "Used to temporarily suppress violent homicidal and suicidal behavior."

```

Comment:
Commas, commas, commas. Use of "and" is okay here but reads better with "or" after commas are inserted.
```suggestion
	desc = "Used to temporarily suppress violent, homicidal, or suicidal behavior in patients."

```